docs/tutorials/autoscaling-federatedhpa-with-cronfederatedhpa.md

```diff
@@ -0,0 +1,189 @@
+kind: FederatedHPA
+metadata:
+  name: nginx-fhpa
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: nginx
+  minReplicas: 1
+  maxReplicas: 10
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 10
+    scaleUp:
+      stabilizationWindowSeconds: 10
+  metrics:
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: 80
+```
+## Deploy CronFederatedHPA in Karmada control plane
+
+To autoscale the minReplicas of FederatedHPA, let's create CronFederatedHPA in the Karmada control plane:
+```yaml
+apiVersion: autoscaling.karmada.io/v1alpha1
+kind: CronFederatedHPA
+metadata:
+  name: nginx-cronfhpa
+  namespace: default
+spec:
+   scaleTargetRef:
+      apiVersion: autoscaling.karmada.io/v1alpha1
+      kind: FederatedHPA
+      name: nginx-fhpa
+   rules:
+   - name: "scale-up"
+     schedule: "*/1 * * * *"
+     targetMinReplicas: 5
+```
+
+The `spec.schedule` follows the following format:
+```
+# ┌───────────── minute (0 - 59)
+# │ ┌───────────── hour (0 - 23)
+# │ │ ┌───────────── day of the month (1 - 31)
+# │ │ │ ┌───────────── month (1 - 12)
+# │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+# │ │ │ │ │                                   7 is also Sunday on some systems)
+# │ │ │ │ │                                   OR sun, mon, tue, wed, thu, fri, sat
+# │ │ │ │ │
+# * * * * *
+```
+The expression `*/1 * * * *` indicates that the FederatedHPA should be scaled every minute.
+
+## Test scaling up
+
+After one minute, check the number of pods to see if the replicas have been scaled up:
+```sh
+$ karmadactl get po
+NAME                     CLUSTER   READY   STATUS    RESTARTS   AGE
+nginx-777bc7b6d7-7vl2r   member1   1/1     Running   0          59s
+nginx-777bc7b6d7-cmt6k   member1   1/1     Running   0          27m
+nginx-777bc7b6d7-pc5dk   member1   1/1     Running   0          59s
+nginx-777bc7b6d7-8lmcg   member2   1/1     Running   0          59s
+nginx-777bc7b6d7-pghl7   member2   1/1     Running   0          59s
+```
+
+And if the business demand is needs more replicas, FederatedHPA will scale the replicas up based on the metrics (.e.g. cpu utilization).  
+
+And checking the CronFederatedHPA's status field, you can see the scaling history:
+```yaml
+apiVersion: autoscaling.karmada.io/v1alpha1
+kind: CronFederatedHPA
+metadata:
+  name: nginx-cronfhpa
+  namespace: default
+spec:
+  rules:
+  - failedHistoryLimit: 3
+    name: scale-up
+    schedule: '*/1 * * * *'
+    successfulHistoryLimit: 3
+    suspend: false
+    targetMinReplicas: 5
+  scaleTargetRef:
+    apiVersion: autoscaling.karmada.io/v1alpha1
+    kind: FederatedHPA
+    name: nginx-fhpa
+status:
+  executionHistories:
+  - nextExecutionTime: "2023-07-29T07:53:00Z"
+    ruleName: scale-up
+    successfulExecutions:
+    - appliedMinReplicas: 5
+      executionTime: "2023-07-29T07:52:00Z"
+      scheduleTime: "2023-07-29T07:52:00Z"
```

Comment:
Add a brief summary of the scaling history?

Add some links to relevant topics at the end?